org.palladiosimulator.analyzer.slingshot.behavior.systemsimulation/src/org/palladiosimulator/analyzer/slingshot/behavior/systemsimulation/SystemSimulationBehavior.java

```diff
@@ -158,9 +165,69 @@ public Result<SEFFInterpretationProgressed> onRepositoryInterpretationInitiated(
 	 * example, when an External Call action was performed.
 	 */
 	@Subscribe
-	public Result<SEFFInterpretationProgressed> onRequestInitiated(
+	public Result<?> onRequestInitiated(
 			final SEFFExternalActionCalled requestInitiated) {
-		final GeneralEntryRequest entity = requestInitiated.getEntity();
+		
+		return requestCallOverWire(requestInitiated.getEntity());
```

Comment:
It troubles me that for a big portion of requests, e.g., calls between colocated assemblies in a resource container, we generate events that require the involvement of the resource simulation behavior instead of completing them within the system simulation behavior.